Tests/MongoSwiftTests/AsyncAwaitTests.swift

```diff
@@ -114,4 +115,168 @@ final class AsyncAwaitTests: MongoSwiftTestCase {
     }
 }
 
```

Comment:
I think it would be nice to have a test for the `deinit` behavior. Basically something that starts up a cursor, doesn't iterate it to completion, and then asserts that a `killCursors` command was eventually sent or something. Ditto for change streams.

Reply:
nice idea! added tests for both.